outputs.tf

```diff
@@ -1,4 +1,4 @@
 output "id" {
   description = "The id of the diagnostic settings."
-  value       = azurerm_monitor_diagnostic_setting.main.id
+  value       = azurerm_monitor_diagnostic_setting.main[*].id
```

Comment:
@larssolem Have you in some way validated that this will work ?

Reply:
Have only validated that it is valid against the terraform modules that are in use

Reply:
It does not look like the output is used. So can probably remove the output